src/core/cwa.ts

```diff
@@ -142,11 +143,22 @@ export default class Cwa {
   getMercureHubURL () {
     const hub = new URL(this.$state.mercureHub)
 
-    for (const resourceType in this.$state.current) {
-      this.$state.current[resourceType].allIds.forEach((id) => {
-        hub.searchParams.append('topic', id)
-      })
+    const appendTopics = (obj) => {
```

Comment:
I'm sure this could be neater. We need to find current IDs twice in this update. Please do feel free to refactor if you see quick improvements